src/pages/LoginPage.tsx

```diff
@@ -82,24 +95,37 @@ function PasswordField(props: PasswordFieldProps) {
   )
 }
 
-function loginWithEmailAndPassword(email, password) {
-  console.log(email + '\n' + password)
-  fetch(buildPath('/helloWorld'), {
-    method: 'POST',
-    headers: { 'Content-Type': 'application/json' },
-  })
-    .then((res) => res.json())
-    .then((res) => {
-      console.log(res.str)
-    })
-}
-
 export default function LoginPage() {
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
 
+  const dispatch = useDispatch()
+  const navigate = useNavigate()
+
+  function loginWithEmailAndPassword(email, password) {
+    fetch(buildPath('/loginWithEmail'), {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ email: email, password: password }),
+    })
+      .then((res: Response) => {
+        if (res.ok) {
+          return res.json()
+        }
+        throw Error(res.statusText)
+      })
+      .then((data) => {
+        console.log(data.userID)
+        dispatch(login(data.userID))
+        navigate('/home', { replace: true })
+      })
+      .catch((error: Error) => {
+        console.log('Login failed: ' + error.message)
```

Comment:
should this be `console.error()`?

Reply:
Not really, because nothing is "going wrong" on the front end if someone types in the wrong email/password.